language/move-prover/stackless-bytecode-generator/src/dataflow_analysis.rs

```diff
@@ -4,12 +4,12 @@
 //! Adapted from AbstractInterpreter for Bytecode, this module defines the data-flow analysis
 //! framework for stackless bytecode.
 
+use crate::stackless_control_flow_graph::StacklessControlFlowGraph;
 use bytecode_verifier::{
     absint::{AbstractDomain, JoinResult},
-    control_flow_graph::{BlockId, ControlFlowGraph},
+    control_flow_graph::BlockId,
```

Comment:
Do we still need this dependency? IIRC it is simply an alias which we can repeat here.

Reply:
Removed